assets/_sass/custom-viz/boeing/map.scss

```diff
@@ -0,0 +1,3 @@
+#map {
```

Comment:
Note that this file should be renamed to `_map.scss`. You do not want to create a standalone `map.css` file.